dlc-manager/src/manager.rs

```diff
@@ -903,6 +905,45 @@ where
         Ok(())
     }
 
+    fn check_preclosed_contracts(&mut self) -> Result<(), Error> {
+        for c in self.store.get_preclosed_contracts()? {
+            if let Err(e) = self.check_preclosed_contract(&c) {
+                error!(
+                    "Error checking confirmed contract {}: {}",
+                    c.signed_contract.accepted_contract.get_contract_id_string(),
+                    e
+                )
+            }
+        }
+
+        Ok(())
+    }
+
+    fn check_preclosed_contract(&mut self, contract: &PreClosedContract) -> Result<(), Error> {
+        let broadcasted_cet = contract
+            .signed_contract
+            .accepted_contract
+            .dlc_transactions
+            .cets[contract.cet_index]
+            .clone();
+        let broadcasted_txid = broadcasted_cet.txid();
+        let confirmations = self
+            .wallet
+            .get_transaction_confirmations(&broadcasted_txid)
+            .unwrap();
+        if confirmations >= 1 {
```

Comment:
Could you use `NB_CONFIRMATIONS` instead of `1`? 6 is usually the number considered safe.